graph/resolvers/asset.js

```diff
@@ -1,4 +1,11 @@
 const Asset = {
+  lastComment({id}, _, {loaders: {Comments}}) {
+    return Comments.getByQuery({
+      asset_id: id,
+      limit: 1,
+      parent_id: null
+    }).then(data => data[0]);
```

Comment:
Also, is there a reason why we can't just use the separate comment query edge instead? Most of these params are exposed here.